examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>Not sure if this is already in the usability drive, but it would be nice to eventually have a high level overview of what's happening under-the-hood here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>